components/infobox/wikis/ageofempires/infobox_league_custom.lua

```diff
@@ -164,8 +165,6 @@ function CustomLeague:defineCustomPageVariables(args)
 	Variables.varDefine('sdate', startdate)
 	Variables.varDefine('edate', enddate)
 
-	Variables.varDefine('game', GameLookup.getName({args.game}))
-	Variables.varDefine('tournament_game', GameLookup.getName({args.game}))
```

Comment:
wouldn't this change the value of the wiki var
and hence the value stored in matches, placements etc.
(might cause some issues on consumer modules)